ekip/everykid/templates/index.html

```diff
@@ -62,15 +62,15 @@
           </div>
           <div class="description">
             <h2>Parents</h2>
-            <p>Whether you stay close to home or travel somewhere new, this year is the perfect time for an unforgettable outing.</p>
+            <p>Stay close to home, or travel somewhere new. This year's the perfect time for an unforgettable outing.</p>
 
-            <a href="{% url 'plan_your_trip' %}"><button>More information</button></a>
+            <a href="{% url 'plan_your_trip' %}"><button>More info</button></a>
```

Comment:
Yeah — using info instead of information bumped the reading level *down* actually. It's more informal for sure.